tests/std/tests/P0896R4_ranges_algorithm_machinery/test.cpp

```diff
@@ -533,3 +533,76 @@ namespace result_test {
     STATIC_ASSERT(test_convertible_3<in_in_out_result>());
     STATIC_ASSERT(test_convertible_3<in_out_out_result>());
 } // namespace result_test
+
+namespace permutable_test {
+    using std::forward_iterator, std::indirectly_movable_storable, std::indirectly_swappable, std::permutable;
+
+    template <int I>
+    struct archetype {
+        using value_type      = int;
+        using difference_type = int;
+
+        struct proxy {
+            proxy()             = default;
+            proxy(proxy const&) = delete;
+            proxy& operator=(proxy const&) = delete;
+
+            operator int() const;
+            // 1: not reflexively indirectly_movable_storable

```

Comment:
Would `// 1: indirectly_movable_storable<archetype, archetype> not satisfied` be clearer here?

Reply:
Yes